src/igvfd/audit/construct_library_set.py

```diff
@@ -75,3 +75,23 @@ def audit_construct_library_set_scope(value, system):
             )
             yield AuditFailure('inconsistent scope metadata',
                                detail, level='WARNING')
+
+
```

Comment:
I suggest to introduce a file with common functions and import the function listing all the "non sequence/configuration" files and use this function in auxiliary and library sets audits, otherwise you introduce code duplication and challenges to maintain